### DISCUSSION TITLE
Enhance UX of contact sidebar

### DISCUSSION THREAD

**Discussion by mrusteberg** [chatwoot/chatwoot#3653] [Feature Requests]
First off, I <3 Chatwoot :)

- remove the edit, merge and delete buttons
-- allow user to edit inline, the edit button is just not very user friendly when copy pasting data out of the actual conversation into the contact
-- display merge button next to merge key if merge key is a match
-- delete is sufficient within contacts list
- move conversation actions into conversation header. it already displays the current assigned user, it would be more intuitive to make that a drop-down to select any user and team
- move Contact Attributes right below Contact Info
- setting to display all contact attributes by default in the sidebar
-- allow to change display order of contact attributes by dragging/dropping attributes in settings
-- remove delete button of contact attributes in sidebar
-- if possible, allow to display contact attributes like default contact info using just icon and value with mouseover alt text

**Comment by nithindavid** [comment]
@mrusteberg Hello, Thanks for the feedback :) Will consider these points for the next iteration.
